Makefile

```diff
@@ -0,0 +1,19 @@
+lint_output  := lint.txt
+
+commit       := $(shell git rev-parse --short HEAD)
```

Comment:
don't need `commit`, `tag` or `version` in the Makefile, none are used as the service doesn't get built to a versioned artefact